---
layout: post
title:  "New preprint! A novel class of carbonic anhydrase-like inorganic carbon transporters"
date:   2018-11-26 13:30:00 -0800
published: true
categories: publications preprints
tags: carbon transporters bacteria chemotrophy
excerpt: I'm pretty excited about this new preprint - Jack Desmarais and I worked on it up until the crack of Thanksgiving. 
---

I'm pretty excited about this new [preprint](https://www.biorxiv.org/content/early/2018/11/22/476713?rss=1) - [Jack Desmarais](https://twitter.com/Jjdesmarais2?lang=en) and I worked on it up until the crack of Thanksgiving. 

The premise of this work is that there are two clearly distinct families of bacterial CO<sub>2</sub> concentrating mechanisms ([CCMs](https://en.wikipedia.org/wiki/Carbon_fixation#CO2_concentrating_mechanisms)) and we wanted to make sure that the mechanistic principles underlying both systems are the same. These families probably evolved convergently and are clearly distinguished by the genes making up the 200+ megadalton [carboxysome structure](https://en.wikipedia.org/wiki/Carboxysome) that forms the centerpiece of the bacterial CCM. One family (&beta;) is found in freshwater cyanobacteria - which have chlorophyll and perform photosynthesis - and the other family (&alpha;) is found in marine cyanobacteria and many proteobacteria that are not photosynthetic but rather [chemotrophic](https://en.wikipedia.org/wiki/Chemotroph). The &beta; family is pretty thoroughly investigated. For example, we think we know all the genes that make up the carboxysome structure and we also know of all the inorganic carbon (Ci) transporters required for the CCM to work. The &alpha; family carboxysome is very well-studied, but until recently it was not clear what transporters are used to power the CCM in proteobacterial chemotrophs that have it. 

![H. neapolitanus](https://genome.jgi.doe.gov/portal/halne/halne.jpg){: .center-image }

Jack started his PhD in the Savage lab by doing a whole-genome knockout screen in *Halothiobacillus neapolitanus* (*Hnea* for short) using a barcoded [TN-Seq](https://en.wikipedia.org/wiki/Transposon_sequencing) approach that was pioneered in the Arkin lab when they were just one floor above us. Their papers using RB-TnSeq are pretty awesome: [1](https://mbio.asm.org/content/6/3/e00306-15.short?rss=1), [2](https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007147), [3](https://www.nature.com/articles/s41586-018-0124-0). Anyway, Jack and Allen Chen (an undergrad-prepostdoc in the Savage lab) isolated 100,000 *Hnea* mutants in our high CO<sub>2</sub> incubator so that we could knockout genes associated with the CCM. 100 thousand mutants! What that means is that the average gene contains ~30 independent knockouts, which has two consequences: 1. if a gene contains 0 knockouts it is almost certainly essential and 2. we have multiple internal biological replicates for the effect of knocking out any individual gene.

All of the transposon mutations contain a random 20 basepair barcode that is unique with high probability. Jack mapped the barcodes to the genome using Illumina sequencing and found that about 70k of the mutants had unique barcodes, i.e. the barcode was found to map to only one genomic locus. Since these barcodes map to only one location, we can safely consider the abundance of the barcode to reflect the abundance of that one mutant in that one location in the *Hnea* genome. So we grow the pooled *Hnea* library in high CO<sub>2</sub> (where the CCM is not required) and ambient CO<sub>2</sub> (where the CCM is required for growth) and compare the abundance of barcodes in those two conditions. Barcodes associated with genes that are required for the CCM should dissapear during growth in ambient CO<sub>2</sub> and not in growth in elevated CO<sub>2</sub>. MOREOVER! Since there are about ~20 unique barcodes per gene, we can see if the individual barcodes associated with a gene produce consistent phenotypes. 

What we found is pleasing, if I do say so myself. First, we built a catalog of genes essential for *Hnea* to grow. This is the first essential gene catalog in a bacterial chemotroph. These genes basically  make sense by eye - all the stuff you'd expect to be essential (the ribosome, DNA replication, central carbon metabolism) is essential. Second - we found that all the known CCM genes, i.e. all the components of the carboxysome and their transcriptional regulators, were essential for CCM function. That is - knockouts of these genes produced a mutant that only grew in elevated CO<sub>2</sub>. This is called a high-CO<sub>2</sub> requiring (HCR) phenotype and is a hallmark of CCM genes. Finally, we found a few genes that had not (at the time we did the screen) been clearly associated with the proteobacterial CCM.

One set of genes now appears to be the missing link in the *Hnea* CCM - the inorganic carbon transporter. The screen showed us that an operon with two components - one homologous to a cation transporter (we call it DabB) and another poorly studied domain of unknown function (DUF2309, we call it DabA) are required for growth in ambient CO<sub>2</sub>. At the time we started this work, little was known about these genes, though impressive recent work from [Kathleen Scott](http://biology.usf.edu/ib/faculty/kscott/) and students has implicated a related family of genes in Ci uptake in proteobacteria (see these papers [4](https://jb.asm.org/content/199/7/e00871-16), [5](https://aem.asm.org/content/early/2018/11/12/AEM.02096-18)). One lucky day Jack decided to run structural homology modeling software on DabA, which he found to be homologous to a bacterial-type &beta; carbonic anhydrase (CA) - an enzyme that accelerates the hydration of CO<sub>2</sub> to HCO<sub>3</sub><sup>-</sup>. With the help of our compatriots Cissi, Eli, Tom and Luke (no particular order) we followed this thread pretty far and showed. 

* DabA and DabB together function in *E. coli* to pump <sup>14</sup>C-labeled Ci into the cell. 
* This complex can be purified from *E. coli* as a single homogeneous structure with no obvious binding partners.
* Treatment of cells with the ionophore CCCP blocks DabAB activity.
* Like the homologous &beta;-CAs, DabA depends on particular histidine, cysteine and aspartic acid residues to function.
* DabA binds a zinc ion, but cysteine mutants of DabA do not (as expected for a &beta;-CAs).
* DabA is not an active CAs in vitro. 

Based on all these results, we infer that the DabAB complex somehow couples the CA active site of DabA to cation (e.g. proton or sodium) transport through DabB. This makes sense since both subunits are required for function, the CA-like active site also appears to be necessary and uncoupling the cation gradients (i.e. CCCP treatment) appears to be required for DabAB to function. This would make DabAB an energetically-activated Ci uptake system. In this model, CO<sub>2</sub> would enter the cell passively and then become hydrated to HCO<sub>3</sub><sup>-</sup> using energy stored in a cation gradient. Since energy-coupled transport is required for the CCM to function, coupling to a cation gradient would enable DabAB to power the *Hnea* CCM. Note that this last bit is our best interpretation of the data we have and not 100% solid - other models could also work. 

We think these results are pretty exciting for a few reasons. First, one of the big open questions about protein complexes, especially multi-subunit protein complexes in membranes like [Complex 1](https://en.wikipedia.org/wiki/Electron_transport_chain#Complex_I), is how activity at one end of the complex (e.g. a redox reaction or CO<sub>2</sub> hydration) is coupled to activity at another end of the complex (e.g. proton or sodium pumping across the membrane). Since the DabAB complex contains only two proteins and appears to perform this kind of long-range coupling, we think it is an exciting model complex. Second, we found that DabAB functions much better in *E. coli* than transporters derived from cyanobacteria. Since pumping inorganic carbon has some important applications in CO<sub>2</sub> sequestration, we think the DAB system might be useful for CO<sub>2</sub> sequestration in the future. Finally, this and other work on Dab-like complexes (e.g. from Kathleen Scott) closes the loop on the proteobacterial CCM, showing that it is in fact quite similar to the cyanobacterial CCM in requiring only a carboxysome and transporters to function. 

Jack (AKA John Desmarais, the first author) really carried the water on this project. Big congrats to him the excellent work. Really happy to be part of this.